### DISCUSSION TITLE
different boxSizes for every nodes

### DISCUSSION THREAD

**Discussion by makobec** [ahnaineh/org_chart#1] [Ideas]
I am currently manually adding a grouping to this org chart but run into a problem where each node has the same box size. I wish we could specify individual box sizes.

**Comment by ali-hnaineh** [comment]
Heyy, yeah currently the box size is common between the nodes, I tried making it more customizable but it was kinda complicated due to how im handling position calculations, I need the boxsize to calculate the position of the nodes, and getting that from the list of widgets widget by widget was a "bit" tricky so i overlooked then... I'd like having that. Maybe i'll soon give it another shot. I'm aiming to make this package as customizable as I can, and allow the devs to style it the way they want!

**Comment by ali-hnaineh** [comment]
May you tell me a usecase for having different boxSizes😁😁
